lib/cosmos/script/script.rb

```diff
@@ -21,6 +21,7 @@
 
 $cmd_tlm_server = nil
 $disconnected_targets = nil
+$disconnect_all_targets = false
```

Comment:
Ultimately why was this necessary? If all targets are disconnected you proxy to the disconnected instance rather than try to figure out if the target was included. If all targets are included I guess this is an optimization but is that all?

Reply:
It was intended to make disconnect mode more reliable, in the nominal case (everything disconnected), the code path is way simpler and will always work.  There are probably still exceptions to the subset of targets cases that don't work.